templates/main.php

```diff
@@ -1 +1,8 @@
-<div id="content"></div>
```

Comment:
well this element is replaced anyway on mounting

I had a failure in my comment. Why actually should be done is:
https://github.com/nextcloud/app-tutorial/blob/master/src/App.vue#L2
Should use `NcContent`

https://nextcloud-vue-components.netlify.app/#/Components/App%20containers?id=nccontent

Reply:
I agree about using `NcContent` - I thought I'd do that in another PR.
But I think the element that is replaced on mounting is actually this:
https://github.com/nextcloud/server/blob/master/core/templates/layout.user.php#L125
It's a parent element so the `div#content` will also be removed.

I'm not sure what vue will do if the id is used twice. What i am trying to do here is to avoid that ambiguity.

Reply:
> But I think the element that is replaced on mounting is actually this:
https://github.com/nextcloud/server/blob/master/core/templates/layout.user.php#L125

Which is exactly why your PR is "wrong". That element must not be removed/replaced.

Reply:
Uh wow... but then we cannot use `NcAppContent` anymore because
> A document must not have more than one [main](https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element) element that does not have the [hidden](https://html.spec.whatwg.org/multipage/interaction.html#attr-hidden) attribute specified.
https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element

Or rather we should turn that `main` element into a div and use a different id then `content` on it.

Reply:
> That element must not be removed/replaced.

But that's already happening anyway. Here's the dom in current master branch before the vue mount:
![Bildschirmfoto vom 2023-02-10 11-13-03](https://user-images.githubusercontent.com/97337118/218065582-000a6fe9-1fc9-4161-826b-27628365547d.png)


And this is the dom afterwards:
![grafik](https://user-images.githubusercontent.com/97337118/218065466-7ccd9ea4-4063-43cc-86fb-60c534026685.png)

Reply:
Hmm right, seems like talk also as an empty template:
https://github.com/nextcloud/spreed/blob/master/templates/index.php

I thought we always nest. But then it seems your current patch is fine.